app/javascript/components/pf_aggregate_status_card.jsx

```diff
@@ -32,8 +32,8 @@ const PfAggregateStatusCard = ({
       </h2>
       <div className="card-pf-body">
         <p className="card-pf-aggregate-status-notifications">
-          { notifications.map(notification => (
-            <span key={notification} className="card-pf-aggregate-status-notification">
+          { notifications.map((notification, i)=> (
```

Comment:
Umm, there's still a missing space :confused: 
```suggestion
          { notifications.map((notification, i) => (
```

If you integrate eslint into your editor, it should complain about this, we have a config for that.
btw @NickLaMuro how is the pronto integration for `miq-bot` going? Because with pronto we could run eslint as well...

Reply:
> btw @NickLaMuro how is the pronto integration for `miq-bot` going?

So, I had https://github.com/ManageIQ/miq_bot/pull/500 as `[WIP]`, and I think the other thing I was waiting on was https://github.com/ManageIQ/miq_bot/pull/505 for this comment:

https://github.com/ManageIQ/miq_bot/pull/500#discussion_r423231344

So I think I need to do some rebasing and change some of the existing code to use this new `GitService` code, but otherwise I think it is mostly good to go.  Will try taking a look at it today.

Reply:
@skateman thanks for the suggestion. I added eslint plugin to my IDE, and added the missing space. I also squashed the commits to a single commit.